scss/forms/_form-check.scss

```diff
@@ -135,8 +135,8 @@
   // Boosted mod: required symbol
   &:required {
     ~ .form-check-label::after {
-      margin-left: $form-label-required-margin-left;
```

Comment:
Is `$form-label-required-margin-left` still used somewhere or should it be deprecated?

Reply:
IMO, we shouldn't remove this rule, I'd maybe have a closer look to know why we have a space in this specific case. Because the other labels look fine.